modules/node_modules/@frogpond/ccc-lib/html.js

```diff
@@ -0,0 +1,25 @@
+import {toLaxTitleCase} from '@frogpond/titlecase'
+
+import _jsdom from 'jsdom'
+const {JSDOM} = _jsdom
+
+export {encode, decode} from 'html-entities'
```

Comment:
How did we handle these here before adding this module? Does JSDOM handle this for us automatically when we call textContent?

Reply:
Great point. Looks like JSDOM handles decoding the entities properly for us. I've [created a Repl][1] to show the differences between `fastGetTrimmedText` and JSDOM's `textContent`.

[1]: https://replit.com/@drewvolz/NaiveMintyArguments#index.js